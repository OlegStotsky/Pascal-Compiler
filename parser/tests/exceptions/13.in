var
a : integer;

type 
b = ^a;