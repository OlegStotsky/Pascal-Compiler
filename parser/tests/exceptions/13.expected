Error at line 4, column 4 : a isn't type
